deployment/src/main/resources/templates/api.qute

```diff
@@ -39,6 +39,9 @@ import io.quarkiverse.openapi.generator.annotations.GeneratedParam;
 @RegisterProvider(CompositeAuthenticationProvider.class)
 @RegisterClientHeaders(AuthenticationPropagationHeadersFactory.class)
 {/if}
+{#for crpClassConfig in custom-register-providers.orEmpty}
```

Comment:
Hello,
Isn't the import missing here ?

Reply:
Hi @YassinHajaj! Thanks for pointing this out. This information came from the property the user configured, so it must be the FQN. There's nothing we can do about it here. See: https://github.com/quarkiverse/quarkus-openapi-generator#custom-register-providers-for-generated-api